forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -38,18 +45,15 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * A method to securely tokenize an EBT card via ForagePANEditText
+     * Tokenizes an EBT Card via a [ForagePANEditText][com.joinforage.forage.android.ui.ForagePANEditText] Element.
```

Comment:
In general, we should document public-facing code to be generic/PaymentMethod.type agnostic – unless we're referring to an EBT-specific thing, but I think it's fine for now to keep the comment here as-is.

Reply:
@djoksimo I will follow up on this in our Linear discussion!